ui/components/app/token-cell/token-cell.js

```diff
@@ -8,7 +8,7 @@ import { TokenListItem } from '../../multichain';
 export default function TokenCell({ address, image, symbol, string, onClick }) {
   const tokenList = useSelector(getTokenList);
   const tokenData = Object.values(tokenList).find(
-    (token) => token.symbol === symbol,
+    (token) => token.symbol === symbol && token.address === address,
```

Comment:
the problem comes from the fact that we filter only on the `symbol`, so this error occurs for two tokens with the same `symbol` but two different addresses, we always take the first in the list of the `getTokenList` result.
the solution is to add the filter with the `address`